pipelines/coaddQualityCore.yaml

```diff
@@ -6,8 +6,12 @@ tasks:
     config:
       bands: ['u', 'g', 'r', 'i', 'z', 'y']
       connections.outputName: objectTableCore
-      plots.shapeSizeFractionalDiffScatter: ShapeSizeFractionalDiffScatter
+      plots.shapeSizeFractionalDiffScatter: ShapeSizeFractionalDiffScatterPlot
```

Comment:
This seems like a good naming convention to add "Plot" since "scatter" is ambiguous.